vm/devices/net/netvsp/src/lib.rs

```diff
@@ -2477,6 +2481,20 @@ impl<T: RingMem> NetChannel<T> {
             });
         }
 
+        if metadata.offload_tcp_segmentation {
+            if segments.len() < 2 {
+                return Err(WorkerError::InvalidLsoPacketInsufficientSegments(
+                    segments.len() as u32,
+                ));
+            }
+            let first_segment_size = segments[0].len;
+            if first_segment_size > 256 {
```

Comment:
This is a MANA requirement, but I dont see this as a (R)NDIS requirement here https://learn.microsoft.com/en-us/windows-hardware/drivers/network/offloading-the-segmentation-of-large-tcp-packets

Reply:
Ok, I'll remove this check.